src/Components/Category/index.jsx

```diff
@@ -18,43 +18,41 @@ function Category() {
     <div>
       <header className="category">
         <Link className="content flex-c" to="/products">
-          <div
-            className="color-box"
-            onClick={() => dispatchFilter({ type: "MEN", payload: !men })}
-          >
-            <img alt="Men" src={Men} />
+          <div onClick={() => dispatchFilter({ type: "MEN", payload: !men })}>
+            <div className="color-box">
+              <img alt="Men" src={Men} />
```

Comment:
in the alt you can directly call the gender from the database.

Reply:
Solved via voice call